typings/blockly.d.ts

```diff
@@ -13422,6 +13436,14 @@ declare module Blockly.Functions {
      * @return {boolean} True if the block is a function argument reporter.
      */
     function isFunctionArgumentReporter(block: Blockly.BlockSvg): boolean;
+
+    /**
+     * Create a flyout, creates the DOM elements for the flyout, and initializes the flyout.
+     * @param {!Blockly.Workspace} workspace The target and parent workspace for this flyout. The workspace's options will
+     *     be used to create the flyout's inner workspace.
+     * @return { flyout: !Blockly.Flyout, flyoutSvg: SVGSVGElement } True if the block is a function argument reporter.
+     */
+    function createFlyout(workspace: Blockly.Workspace): void;
```

Comment:
missing "return" type? did you use "gulp generate-dts" to regen blockly.d.ts?

Reply:
Looks like generate-dts cannot handle record types. I'll find a workaround.